extensions/wikia/StaffLog/StaffLog_body.php

```diff
@@ -160,32 +158,31 @@ function formatRow($result) {
 				}
 				$out = wfMessage( 'stafflog-blockmsg' ,
 					array($time,
-						$linker->userLink($result->slog_user, User::getUsername( $result->slog_user, $result->slog_user_name ) ),
-						$linker->userLink($result->slog_userdst, $result->slog_user_namedst),
+						Linker::userLink($result->slog_user, User::getUsername( $result->slog_user, $result->slog_user_name ) ),
+						Linker::userLink($result->slog_userdst, User::getUsername( $result->slog_userdst, $result->slog_user_namedst ) ),
 						$siteurl,
 						strlen($result->slog_comment) > 0 ? $result->slog_comment:"-" ))->text();
 				break;
 			case  'piggyback':
 				$msg = $result->slog_action == "login" ? "stafflog-piggybackloginmsg" : "stafflog-piggybacklogoutmsg";
 				$out = wfMessage( $msg,
 					array($time,
-						$linker->userLink($result->slog_user, User::getUsername( $result->slog_user, $result->slog_user_name ) ),
-						$linker->userLink($result->slog_userdst, $result->slog_user_namedst)))->text();
+						Linker::userLink($result->slog_user, User::getUsername( $result->slog_user, $result->slog_user_name ) ),
+						Linker::userLink($result->slog_userdst, User::getUsername($result->slog_userdst, $result->slog_user_namedst ) )
+					)
+				)->text();
 				break;
 			case 'wikifactor':
 				$out = $time . ' ' . $result->slog_comment;
 				break;
 			default:
 				$out = "";
-				Hooks::run('StaffLog::formatRow',array($result->slog_type,$result,$time,$linker,&$out));
+
+				// TODO: used by UserRenameToolStaffLogFormatRow only, remove when we get rid of Special:RenameUser
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this "TODO" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)